invisible_cities/evm/nh5.py

```diff
@@ -29,24 +29,40 @@ class DataSensor(tb.IsDescription):
     adc_to_pes = tb.Float32Col(pos=3)
     noise_rms  = tb.Float32Col(pos=4)
 
+class MCExtentInfo(tb.IsDescription):
+    """Stores. the parameters used by the simulation as metadata using
+    Pytables.
+    """
+    evt_number    = tb.Int16Col(pos=0)
+    last_hit      = tb.Int16Col(pos=1)
+    last_particle = tb.Int16Col(pos=2)
 
-class MCTrack(tb.IsDescription):
+class MCHitInfo(tb.IsDescription):
+    """Stores. the parameters used by the simulation as metadata using
+    Pytables.
+    """
+    hit_position  = tb.Float32Col(    pos=0, shape=3)
```

Comment:
I understand, but I would keep it this way, since we have already a tag in nexus that is being used. Also, the `hit_indx` attribute is better placed after `label`, because it is the ordered number of a hit *with a particular label*.